### ISSUE TITLE
Autosave doesn't work

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/ads-ioc#21]
https://github.com/pcdshub/ads-ioc/blob/c4720149a1f0af1a995646a621324e5cc7a81176/iocBoot/templates/st.cmd.template#L115 changes to the wrong top

https://github.com/pcdshub/ads-ioc/blob/c4720149a1f0af1a995646a621324e5cc7a81176/iocBoot/templates/st.cmd.template#L133 references a file that is never built

**Comment by klauer** [comment]
(1) We're calling "IOC_TOP" where the `st.cmd` is - probably a bad name. Or do you mean it didn't even `cd` there?
(2) `makeAutosaveFiles` needs to be called

**Comment by ZLLentz** [comment]
Expanding on (1): here we need to cd into the directory that has:
```
dbLoadRecords("db/iocSoft.db", "IOC={{prefix}}")
dbLoadRecords("db/save_restoreStatus.db", "P={{prefix}}{{delim}}")
```
Which is the ads module top

(2) is missing from the default `make` I think

**Comment by ZLLentz** [comment]
closed by #22
